src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/FlowMonitoringTopology.java

```diff
@@ -159,7 +161,8 @@ private void zooKeeperBolt(TopologyBuilder topology) {
                 getBoltInstancesCount(ComponentId.ISL_CACHE_BOLT.name(), ComponentId.FLOW_CACHE_BOLT.name()));
```

Comment:
Please add ACTION_BOLT into getBoltInstancesCount. We need  it to calculate expected_state for topology

Reply:
Fixed.